alacritty/src/window_context.rs

```diff
@@ -50,6 +51,8 @@ pub struct WindowContext {
     font_size: Size,
     mouse: Mouse,
     dirty: bool,
+    /// Initial information to identify the window such as default class and title.
```

Comment:
```suggestion
```

None of the other fields are documented, so I think it's odd to explicitly document this one. Especially because it's not particularly difficult to understand.

I have to say though `WindowContext` is getting **big**, I guess that's to be expected though since it holds everything important for a window. Maybe breaking it down in the future into separate structs might make sense, but don't think that's what we should do now.